config/webpack/environment.js

```diff
@@ -1,7 +1,14 @@
 const { environment } = require('@rails/webpacker')
 const { VueLoaderPlugin } = require('vue-loader')
-const vue = require('./loaders/vue')
+const vue =  require('./loaders/vue')
```

Comment:
Multiple spaces found before 'require'  no-multi-spaces<br>'require' is not defined                no-undef<br>Missing semicolon                       semi